pystiche_papers/li_wand_2016/_loss.py

```diff
@@ -105,6 +124,36 @@ def style_loss(
     target_transforms: Optional[Iterable[transforms.Transform]] = None,
     score_weight: Optional[float] = None,
 ) -> ops.MultiLayerEncodingOperator:
+    r"""Style_loss from :cite:`LW2016`.
+
+    Args:
+        impl_params: If ``True``, uses the parameters used in the reference
+            implementation of the original authors rather than what is described in
+            the paper. For details see below.
+        multi_layer_encoder: Pretrained :class:`~pystiche.enc.MultiLayerEncoder`. If
+            omitted,  the default
+            :func:`~pystiche_papers.li_wand_2016.multi_layer_encoder` is used. Defaults
+            to ``None``.
+        layers: Layers from which the encodings of the ``multi_layer_encoder`` should be
+            taken. If omitted, the defaults is used. Defaults to
+            ``("relu3_1", "relu4_1")``.
+        layer_weights: Layer weights of the operator. Defaults to ``"sum"``.
+        patch_size: Size of the patch.
+        stride: Stride of the convolution. If omitted, the stride is determined with
+            respect to `impl_params``. For details see below.
+        target_transforms: Optional augemntation transformations for the target. If
+            omitted, the transfomrms are determined with respect to `impl_params``. For
+            details see below.
+        score_weight: Score weight of the operator. If omitted, the score_weight is
+            determined with respect to `impl_params``. For details see below.
+
+    If ``impl_params is True`` , a stride of 2 is used instead of 1, a score_weight of
+    1e-4 instead of the 1e0, and an additional score correction factor of 1.0 / 2.0 is
+    used instead of none. Additionally, no transformations are used in the scaling and
```

Comment:
The transforms are used regardless of `impl_params`. Only their parameters change.

Reply:
Yes, but with `impl_param is True` the number of transformations is set to 1, so no additional transformation takes place. Or am I misunderstanding this right now?

Reply:
Where do you get this from?

Reply:
From the original implementation. See [here](https://github.com/pmeier/CNNMRF/blob/fddcf4d01e2a6ce201059d8bc38597f74a09ba3f/cnnmrf.lua#L52) for the number of num_scales. For this reason the `num_scale_steps=1` if` impl_params is True` in this replication.

Reply:
With `num_scale_steps == 1` you will have three transformations. Nonetheless, the reference implementation seems to default to `0` steps. I got the parameters from 

https://github.com/pmeier/CNNMRF/blob/fddcf4d01e2a6ce201059d8bc38597f74a09ba3f/run_trans.lua#L65-L70

since this result is included in the paper. Thoughts?

Reply:
I think we should change this to `0`, as the author explicitly specifies the default value as `0`.
See  [here](https://github.com/pmeier/CNNMRF/blob/fddcf4d01e2a6ce201059d8bc38597f74a09ba3f/run_trans.lua#L17).

Reply:
I think this OK for now, but we should keep this in mind for the replication. Could you send a fix?

Reply:
Just to be on the safe side, do you mean a fix by setting the default value to `0` and the description here in the doc is ok for now?

Reply:
Correct. For the replication though, we should use `num_scale_steps = num_rotation_steps = 1` to get closer to the result in the original replication.